pkg/services/sqlstore/temp_user_test.go

```diff
@@ -62,7 +62,21 @@ func TestTempUserCommandsAndQueries(t *testing.T) {
 
 				So(err, ShouldBeNil)
 				So(query.Result[0].EmailSent, ShouldBeTrue)
-				So(query.Result[0].EmailSentOn, ShouldHappenOnOrAfter, (query.Result[0].Created))
+				So(query.Result[0].EmailSentOn.UTC(), ShouldHappenOnOrAfter, (query.Result[0].Created.UTC()))
+			})
+
+			Convey("Should be able expire temp user", func() {
+				cmd4 := models.ExpireTempUsersCommand{OlderThan: timeNow()}
```

Comment:
Why is it called `cmd4`, as opposed to e.g. `cmd`?

Reply:
Ah yes I was confused about this originally too but seems to be a pattern followed in the preceding tests. There's an original `cmd` for setting up the TempUser and then subsequent nested tests could be changed to something more meaningful / less distracting. WDYT?

Reply:
You probably have the right idea :) My main concern I guess is that the naming is confusing.